### PR TITLE
Changed MDC icon to the new one

### PR DESCRIPTION
**Pull request by ziccardi**
Changed MDC icon

### CODE REVIEW

**Review by wei-lee — CHANGES_REQUESTED** [review]
please check this fie: https://github.com/aerogear/mobile-services-installer/blob/master/apb-release.md to see the steps you need to go through to update the apb.

**Comment by ziccardi** [comment]
@wei-lee mind giving another look?

File: apb.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ tags:
 metadata:
   displayName: Mobile Developer Console
   longDescription: "An APB deploying the AeroGear mdc-server Server with persistent storage and authentication through Openshift"
```
Comment (wei-lee) [inline apb.yml]: can we update the description here to remove `AeroGear`, and change `mdc-server` to `Mobile Developer Console`

Comment (wei-lee) [reply apb.yml]: can we also change the description in L3 as well, Just call it `Mobile Developer Console`

Comment (wei-lee) [reply apb.yml]: actually, there isn't persistent storage here. might just remove it.

Comment (ziccardi) [reply apb.yml]: 👍 Done!

**Review by wei-lee — APPROVED** [review]

**Pull request merged**